src/usdb_syncer/gui/__init__.py

```diff
@@ -73,6 +80,38 @@ def _run() -> None:
     app.exec()
 
 
+def get_latest_version() -> str | None:
+    url = "https://api.github.com/repos/bohning/usdb_syncer/releases/latest"
+    response = requests.get(url, timeout=5)
+    if response.status_code == 200:
+        return response.json()["tag_name"]
+    return None
+
+
+def get_installed_version() -> str | None:
+    try:
+        return pkg_resources.get_distribution("usdb_syncer").version
+    except pkg_resources.DistributionNotFound:
+        return None
+
+
+def check_for_update() -> None:
+    latest_version = get_latest_version()
+    installed_version = get_installed_version()
+
+    if latest_version and installed_version:
+        if version.parse(installed_version) < version.parse(latest_version):
+            logging.warning(
+                f"USDB Syncer {latest_version} is available! "
+                f"(You have {installed_version}). Please download the latest release "
+                "from https://github.com/bohning/usdb_syncer/releases/latest."
```

Comment:
It might be nice to have a visual indicator that could also be a clickable link somewhere in the GUI. As we know, the log is currently a bit overwhelming.

Reply:
I added a hyperlink label next to the search field that is only visible if there is a new version available (and if the app is a bundle).